SPRINT_MISSION_3/app.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import dotenv from 'dotenv';
+
+import productRoutes from './routes/products.routes.js';
+import articleRoutes from './routes/Article.routes.js';
+import commentRouter from './routes/comment.routes.js';
+import errorHandler from './middlewares/errorHandler.js';
+import uploadRouter from './routes/upload.routes.js';
+
+dotenv.config();
+
+const app = express();
+app.use(express.json());
+
+// 라우터 등록
+app.use('/products', productRoutes);
+app.use('/article', articleRoutes);
+app.use('/api', commentRouter);
+app.use(errorHandler);
+app.use('/upload', uploadRouter);
```

Comment:
route path의 일관성이 부족.
보통 RESTFul하게 사용한다고 하면 아래처럼 사용하면 될 것 같아요.
```suggestion
app.use('/products', productRoutes);
app.use('/articles', articleRoutes);
app.use('/comments', commentRouter);
app.use(errorHandler);
app.use('/uploads', uploadRouter);
```